src/main/java/duke/commands/AddCommand.java

```diff
@@ -0,0 +1,85 @@
+package duke.commands;
+
+import duke.exceptions.DukeException;
+import duke.storage.Task;
+import duke.storage.TaskList;
+import duke.ui.Ui;
+
+/**
+ * The AddCommand class defines a command to add tasks to be stored in the Duke
+ * application
+ *
+ * @author Ryan NgWH
+ */
+public class AddCommand extends Command {
```

Comment:
Perhaps a more descriptive name would be AddTaskCommand, since in the future there may be other things you could be adding.